lapce-ui/src/svg.rs

```diff
@@ -71,12 +89,15 @@ pub fn file_svg(path: &Path) -> Svg {
                     (&["js"], "file_type_js.svg"),
                     (&["ts"], "file_type_ts.svg"),
                     (&["css"], "file_type_css.svg"),
+                    (&["svg"], "file_type_svg.svg"),
+                    (&["cmake"], "file_type_cmake.svg"),
+                    (&["make"], "file_type_make.svg"),
                 ];
 
                 for (exts, file_type) in TYPES {
                     for ext in exts.iter() {
                         if extension.eq_ignore_ascii_case(ext) {
-                            return Some(*file_type);
+                            Some(*file_type);
```

Comment:
Why remove the return?

Reply:
I got a issue about it when I submit the pr the first time. Something about it not following the style guide.